src/yb/util/wait_state.h

```diff
@@ -112,6 +112,8 @@ struct AUHMetadata {
   int64_t current_request_id = 0;
   std::string client_node_ip;
 
+  void SetQueryIdFromHex(std::string queryId);
```

Comment:
can become a normal SetQueryId(uint )
which can be called as 
SetQueryId(stoi("foo", 0, 16);